gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -1944,6 +2010,11 @@ def main(args):
 
     mt = filter_genotypes(mt)
     mt = add_variant_annotations(mt, min_hom_threshold)
+    mt = mt.checkpoint(
```

Comment:
Add comment about reason for checkpointing

Reply:
For checkpoints I also sometimes just default to `overwrite=True` since it won't overwrite the actual output, but if I forget to use the  --overwrite flag, at least I can start from the checkpoint. This is a personal preference so I leave it completely up to you